modules/express/test/unit/bitgoExpress.ts

```diff
@@ -203,6 +203,25 @@ describe('Bitgo Express', function () {
       logStub.restore();
     });
 
+    it('should output external mode upon server startup', () => {
```

Comment:
could use a test to make sure we only call `setupAPIRoutes()` when running in regular mode (and that `setupSigningRoutes()` is not called), and another test for the inverse case